tests/unit/components/plate_based_widgets/mapeditor/PlateMapHover.spec.js

```diff
@@ -0,0 +1,415 @@
+import { mount } from "@vue/test-utils";
+import ComponentToTest from "@/components/plate_based_widgets/mapeditor/PlateMapEditor.vue";
+// import { shallowMount } from "@vue/test-utils";
+
+import Vuex from "vuex";
+import { createLocalVue } from "@vue/test-utils";
+
+let wrapper = null;
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+let NuxtStore;
+let store;
+
+describe("PlateMapEditor.vue", () => {
+  beforeAll(async () => {
+    // note the store will mutate across tests, so make sure to re-create it in beforeEach
+    const storePath = `${process.env.buildDir}/store.js`;
+    NuxtStore = await import(storePath);
+  });
+
+  beforeEach(async () => {
+    store = await NuxtStore.createStore();
+  });
+
+  afterEach(() => wrapper.destroy());
+  test("Given that none of the wells are selected, minus button should not be visible and stroke outlines should be zero on all wells, When hovers the plus button, Then all 24 wells should have a stroke hover outline", async () => {
```

Comment:
It looks like in the body of the test you are also asserting about what happens when the user stops hovering. please include that in the test title

Reply:
Now the title has complete details of the test.
`Given that none of the wells are selected, minus button should not be visible and stroke outlines should be zero on all wells, When user hovers the plus button and later un hovers the plus button, Then all 24 wells should have a stroke hover outline of 2px during hover and stroke hover outline is reset to 0px when un-hover the plus button`